SBOMDocumentTypes.md

```diff
@@ -0,0 +1,30 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# SBOMDocumentTypes
+
+## Summary
+
+Provides a set of values to be used to describe the common types of SBOMs that tools may create.
+
+## Description
+
+This field provides a list with agreed on definitions of SBOM types, as defined in [Types of Software Bill of Material (SBOM) Documents](https://www.cisa.gov/sites/default/files/2023-04/sbom-types-document-508c.pdf), published on April 21, 2023. 
```

Comment:
If the field name is singular per @armintaenzertng's comment then why is this field described as a list? Shouldn't it be only a single value (from the list defined in https://www.cisa.gov/sites/default/files/2023-04/sbom-types-document-508c.pdf?).